plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcColumnHandle.java

```diff
@@ -152,6 +152,16 @@ public long getRetainedSizeInBytes()
                 + jdbcTypeHandle.getRetainedSizeInBytes();
     }
 
+    public static JdbcColumnHandle createJdbcColumnHandle(String columnName,
+                                                          JdbcTypeHandle jdbcTypeHandle,
```

Comment:
I don't think that wrapping here improves the readability. Please make it a single line

Reply:
fixed